### PR TITLE
docs: add warning about not adding package.json

### LINKED ISSUE #969: [DOCS] Add warning about not adding package.json To contributing.md
### Description

Many people accidentally add and commit their package.json and package-lock.json files when contributing their profile. I suggest making the contribution guidelines clearer, maybe with a warning not to add these files and not to use git add . 

It could be added somewhere here in the contributing.md file:
286BDA27-C849-423A-A31D-4EE40826EC35

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Hat52**
resolve #969 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/976"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Please remove your profile from the pull request. That should solve the merge conflict.

File: public/data/HAT52.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (krishguptadev) [inline public/data/HAT52.json:1]: Please move this to another PR?

File: Contributing.md [hunk 2]
```diff
@@ -57,6 +57,10 @@ git merge upstream/main
 
 **8.** Stage your changes and commit:
 
+⚠️ **Make sure** not to commit package.json or package-lock.json file
+
+⚠️ **Make sure** not to run the command `git add .` or `git add *`
```
Comment (krishguptadev) [inline Contributing.md:62]:
```suggestion
⚠️ **Make sure** not to run the command `git add .` or `git add *` instead stage your changes for each file/folder
```

File: Contributing.md [hunk 3]
```diff
@@ -57,6 +57,10 @@ git merge upstream/main
 
 **8.** Stage your changes and commit:
 
+⚠️ **Make sure** not to commit package.json or package-lock.json file
+
+⚠️ **Make sure** not to run the command `git add .` or `git add *`instead stage your changes for each file/folder
```
Comment (Panquesito7) [inline Contributing.md:62]:
```suggestion
⚠️ **Make sure** not to run the commands `git add .` or `git add *`. Instead, stage your changes for each file/folder
```

File: Contributing.md [hunk 4]
```diff
@@ -57,6 +57,10 @@ git merge upstream/main
 
 **8.** Stage your changes and commit:
 
+⚠️ **Make sure** not to commit package.json or package-lock.json file
```
Comment (Panquesito7) [inline Contributing.md:60]:
```suggestion
⚠️ **Make sure** not to commit `package.json` or `package-lock.json` file
```

**Comment by Hat52** [comment]
Are there any issues with this pull request?
I have removed my profile from the pull request.

**Comment by emmalearnscode** [comment]
There are a couple of suggestions from @Panquesito7  which help to make it more readable. If you could make these small updates this should be good to go 👍🏻

**Comment by Hat52** [comment]
Ok

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
Could you please take a look at my suggestion https://github.com/EddieHubCommunity/LinkFree/pull/976/files#r783273119 and let me know what do you think? Thanks. 🙂

**Review by krshdev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**